### PR TITLE
Add Shirley Yu leaderboard submission

### PR DESCRIPTION
**Pull request by ShirleyU01**
Shirley Yu 
4.045071189403534
 https://wandb.ai/weiranyu-stanford-university/cs336-tinystories/runs/znwy1mq5?nw=nwuserweiranyu

### CODE REVIEW

**Comment by hermabr** [comment]
* Make sure your link is shared with everyone (check if it works in incognito)
* The url mentions tinystory, but the leaderboard is for owt. Which did you run on?
* Please add a description of what you did

**Comment by ShirleyU01** [comment]
Hi Herman!

- I tried to make it public, but for some reason wandb only allow me to share with team. Here is the plots and screenshots of run info
owt_val (1)
owt_train (1)
Screenshot 2026-04-17 at 5 36 53 PM
Screenshot 2026-04-17 at 5 37 04 PM

Please let me do if those are not sufficient. Thank you!

- This run was on owt. I named the workspace as cs336-tiny in the beginning and forgot to change it.

- I fine-tuned the learning rate and then used the one worked the best (--max_lr 0.001 --min_lr 0.0001) to perform a full owt run, then I found out the result is better than the baseline. I was ran out of credit so I did not do too many modification/experiments on owt.

**Comment by marcelroed** [comment]
Can you upload the val_loss curve to the `images/` directory and link it instead? Thanks!

**Comment by ShirleyU01** [comment]
Hi Marcel! Here is the link for val_loss curve: https://github.com/ShirleyU01/assignment1-basics-leaderboard/blob/main/images/shirleyu01.png, and I have changed the link in README.md and pushed it as well. Thanks!

**Comment by hermabr** [comment]
Great! It seems like .DS_Store was accidentally added to the pr. That should be the final thing before your pr is ready to merge:))

**Comment by ShirleyU01** [comment]
Just fixed it! Thanks for your help on the leaderboard submission :))
